Assets/Project/Modules/PlayerAnchor/Scripts/Player/View/SquashStretchView/PlayerSquashAndStretchView.cs

```diff
@@ -78,11 +78,11 @@ public void PlayHealAnimation()
             PunchScale(_config.HealScalePunch);
         }
 
-        public void PlayDashAnimation(float duration)
+        public void PlayDashAnimation(float duration, Vector3 dashDirection)
         {
             ClearTweens();
-            PunchScale(_config.DashScalePunch, duration);
-            PunchRotate(_config.DashRotationPunch, duration);
+            //PunchScale(_config.DashScalePunch, duration);
```

Comment:
Same as the previous comment. It would be better to use something like //MIGHT NEED: "code"